run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
do we need this at the top-level of the repo?

Reply:
no we don’t. some random editor extension auto added it. thanks for spotting.